src/justfile.rs

```diff
@@ -395,6 +396,10 @@ impl<'src> Justfile<'src> {
     recipes
   }
 
+  pub(crate) fn groups(&self) -> Vec<String> {
```

Comment:
This can return `&[String]`, so that we can avoid a clone of the whole vec.

Reply:
Updated.